src/catalog/column_catalog.cpp

```diff
@@ -15,35 +15,38 @@
 #include "catalog/catalog.h"
 #include "catalog/system_catalogs.h"
 #include "catalog/table_catalog.h"
+#include "codegen/buffering_consumer.h"
 #include "concurrency/transaction_context.h"
 #include "storage/data_table.h"
 #include "storage/database.h"
 #include "type/value_factory.h"
+#include "expression/expression_util.h"
 
 namespace peloton {
 namespace catalog {
 
-ColumnCatalogObject::ColumnCatalogObject(executor::LogicalTile *tile,
-                                         int tupleId)
-    : table_oid(tile->GetValue(tupleId, ColumnCatalog::ColumnId::TABLE_OID)
+
+ColumnCatalogObject::ColumnCatalogObject(codegen::WrappedTuple wrapped_tuple)
+    : table_oid(wrapped_tuple.GetValue(ColumnCatalog::ColumnId::TABLE_OID)
```

Comment:
As mentioned by @gandeevan, GetValue is deprecated now. Please talk to @poojanilangekar to make sure you can use this for a wrapped tuple object safely